package.json

```diff
@@ -15,6 +15,7 @@
     "typecheck": "tsc"
   },
   "dependencies": {
+    "@deriv/ui": "^0.0.15",
```

Comment:
can you please install the `0.1.0` version? 
this version contains the SSR issue fix. the `0.0.15` version works in `dev` env. but on build time it breaks. 
this is fixed in `0.1.0` thanks :pray:

Reply:
Done 👍 I also made a local build and served/tested it. Looks like everything seems to be working fine without breaking anything.